src/saltext/vmware/modules/esxi.py

```diff
@@ -2059,3 +2059,122 @@ def get(
         return ret
     except DEFAULT_EXCEPTIONS as exc:
         raise salt.exceptions.SaltException(str(exc))
+
+
+def _get_host(host, service_instance=None):
```

Comment:
I would put this in utils/esxi.py

Reply:
is uses `__opts__` which utils/esxi does not get